src/inference/src/dev/core_impl.cpp

```diff
@@ -1248,17 +1249,8 @@ ov::AnyMap ov::CoreImpl::create_compile_config(const ov::Plugin& plugin, const o
         // for the default device (e.g. DEVICE_ID = 0 for GPU)
     }
 
-    // 2. Replace DEVICE_ID with DEVICE_ARCHITECTURE value to identify device
-    if (device_supports_property(plugin, ov::device::architecture.name())) {
-        compile_config[ov::device::architecture.name()] =
-            plugin.get_property(ov::device::architecture, property_config);
-    } else {
-        // Take device name if device does not support DEVICE_ARCHITECTURE metric
-        compile_config[ov::device::architecture.name()] = plugin.get_name();
-    }
-
-    // 3. Extract config keys which affect compilation process
-    if (device_supports_property(plugin, ov::caching_properties.name())) {
+    // 2. Extract config keys which affect compilation process
+    if (device_supports_property(plugin, ov::caching_properties)) {
```

Comment:
I think here is very clear!
It need all HW plugins provide ov::caching_properties supported, currently CPU/GPU/GNA all support, but new HW plugin should it if have.

Reply:
fixed